src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -1201,5 +1203,69 @@ uint ITypesDebugInfoWriter.GetMemberFunctionId(MemberFunctionIdTypeDescriptor me
         {
             return GetMemberFunctionIdTypeIndex(_nativeObjectWriter, memberIdDescriptor);
         }
+
+        private static string GetLLVMTripleFromTarget(TargetDetails target)
+        {
+            // We create a triple based on the Target
+            // Detect the LLVM arch
+            string arch = null;
```

Comment:
This does not need to be set to null - to make it clear that it is always set below.

Reply:
Fixed by commit 2e59964